FE/src/components/Bottomsheet/KidInfoBox/KidInfoBox.jsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./kidInfoBox.module.scss";
+import { ContentsBox } from "../ContentsBox/ContentsBox";
+
+export function DriverContents({ childrenData }) {
+    const isHaveData = childrenData.length > 0 ? true : false;
```

Comment:
```suggestion
const isHaveData = childrenData.length > 0;
```
이렇게 해도 똑같이 boolean으로 쓸 수 있어요!